src/plugin/weekOfYear/index.js

```diff
@@ -12,4 +15,7 @@ export default (o, c, d) => {
     const diffInWeek = this.diff(compareDay, W, true)
     return Math.ceil(diffInWeek)
   }
+  proto.weeks = function (week = 0) {
```

Comment:
Be aware that, if user call `weeks` alias, without arguments, the default value `week = 0` will set the week to 0. You should change the default value to `null`, the same for `week()`

```javascript
proto.weeks = function (week = null) {
    return this.week(week)
}
```